node_modules/col-whiteboards/tests/util.js

```diff
@@ -306,26 +352,82 @@ var assertGetWhiteboardsFails = module.exports.assertGetWhiteboardsFails = funct
  * @param  {String}             title                           The updated title of the whiteboard
  * @param  {Number[]}           members                         The ids of the users that should be a member of the whiteboard
  * @param  {Function}           callback                        Standard callback function
- * @param  {Whieboard}          callback.whiteboard             The updated whiteboard
+ * @param  {Whiteboard}         callback.whiteboard             The updated whiteboard
  * @throws {AssertionError}                                     Error thrown when an assertion failed
  */
 var assertEditWhiteboard = module.exports.assertEditWhiteboard = function(client, course, id, title, members, callback) {
-  client.whiteboards.editWhiteboard(course, id, title, members, function(err, whiteboard) {
-    assert.ok(!err);
-    assert.ok(whiteboard);
-    assertWhiteboard(whiteboard, {'expectFullWhiteboard': true});
-    assert.strictEqual(whiteboard.title, title);
+  UsersTestUtil.assertGetMe(client, course, null, function(me) {
 
-    // Verify that the expected members are present
-    var expectedMembers = CollabosphereUtil.toArray(members);
-    expectedMembers = _.uniq(expectedMembers);
+    // Get the whiteboard in its current state
+    assertGetWhiteboard(client, course, id, null, null, function(oldWhiteboard) {
+      var oldMemberIds = _.pluck(oldWhiteboard.members, 'id');
 
-    assert.strictEqual(whiteboard.members.length, expectedMembers.length);
-    _.each(expectedMembers, function(member) {
-      UsersTestUtil.assertUser(_.findWhere(whiteboard.members, {'id': member}));
-    });
+      var expectedMembers = CollabosphereUtil.toArray(members);
+      expectedMembers = _.uniq(expectedMembers);
 
-    return callback(whiteboard);
+      var newMemberIds = _.difference(expectedMembers, oldMemberIds);
+
+      // Collect any emails that are sent out when editing the whiteboard
+      var emails = [];
+      function emailListener(email, user, course) {
+        emails.push({
+          'email': email,
+          'user': user,
+          'course': course
+        });
+      };
+      EmailUtil.on(EmailUtil.EVENT_NAMES.EMAIL_SENT, emailListener);
+
+      // Because of the asynchronous process of sending out emails, we only return to the caller when
+      // both the REST request finishes and the emails have been sent out
+      var _whiteboard = null;
+      var done = _.after(2, function() {
```

Comment:
Same here. Shouldn't this expect a number of emails equal to the number of new members ?